ui/src/router/activity_log/RouterActivityLogView.js

```diff
@@ -32,21 +32,16 @@ export const RouterActivityLogView = ({ projectId, routerId, router }) => {
   );
   const hasInitiallyLoaded = useInitiallyLoaded(isLoaded);
 
-  const routerStatus = useMemo(
-    () => Status.fromValue(router.status),
-    [router.status]
-  );
-
   useEffect(() => {
     fetchEventsOnce();
   }, [fetchEventsOnce]);
 
   useEffect(() => {
-    if (routerStatus === Status.PENDING) {
+    if (router.status === Status.PENDING) {
```

Comment:
Such changes are required here and in other places because, the router / version objects are now the parsed objects rather than the raw response. So, for example, the status property of the router here is not merely a string, but an Enum object ([ref](https://github.com/gojek/turing/blob/main/ui/src/services/router/TuringRouter.js#L61)).